setup.py

```diff
@@ -26,10 +26,11 @@
         'cdo',
         'cf-units',
         'cython',
-        'jinja2',
+        'ecmwf-api-client',
```

Comment:
Please also add the conda package in meta.yaml or the conda build will break

Reply:
@bouweandela Thanks, I added the  'ecmwf-api-client' to meta.yaml